packages/site/src/components/Abstract.tsx

```diff
@@ -3,10 +3,12 @@ import { ContentBlocks } from './ContentBlocks';
 import classNames from 'classnames';
 
 export function Abstract({ content }: { content: GenericParent }) {
-  if (!content) return null;
+  if (!content) return <div className="hidden" aria-label="this article has no abstract" />;
   return (
     <>
-      <span className="mb-3 font-semibold">Abstract</span>
+      <span className="mb-3 font-semibold" tabIndex={0}>
+        Abstract
+      </span>
```

Comment:
There is a good walkthrough video of ar5iv here that argues for abstract to be a (styled) h2 element.

https://github.com/brucemiller/LaTeXML/issues/2066#issuecomment-1696389753

By Patrick Smyth from the Iota School on YouTube:
https://youtu.be/bHM6mes51nE

Was worth a watch for me, I think this PR is the major thing to implement.

Reply:
I changed to an h2!